spec/integration/endpoints/logs_spec.rb

```diff
@@ -8,19 +8,20 @@ module Travis::GuestApi
     include Rack::Test::Methods
 
     def app
-      Travis::GuestApi::App.new(1, reporter, &callback)
+      Travis::GuestApi::App.new(job_id, reporter, &callback)
     end
 
+    let(:job_id) { nil }
     let(:reporter) { double(:reporter) }
     let(:callback) { ->(x) { } }
 
     describe 'POST /logs' do
-      let(:post_data1) { { job_id: 1, message: 'my message1' } }
-      let(:post_data2) { { job_id: 1, message: 'my message2' } }
+      let(:post_data1) { { job_id: 123, message: 'my message1' } }
+      let(:post_data2) { { job_id: 456, message: 'my message2' } }
 
       it 'sends data to pusher' do
```

Comment:
it "sends data to **reporter**"
(we do not use pusher at all, slanger is called from travis-logs). Reporter class sends data to RabbitMQ. Travis-logs read this queue.